test/init.bats

```diff
@@ -35,12 +35,12 @@ create_executable() {
   assert_line "command pyenv rehash 2>/dev/null"
 }
 
-
 @test "setup shell completions" {
   root="$(cd $BATS_TEST_DIRNAME/.. && pwd)"
+  export PYENV_ROOT=$root
```

Comment:
Same as above, do not confuse the two roots.

Reply:
The reason why I am setting this, is because the original PYENV_ROOT (which I guess is determined by the machine which runs the tests) is overridden by `test/test_helper.bash`

```bash
if [ -z "$PYENV_TEST_DIR" ]; then
  PYENV_TEST_DIR="${BATS_TMPDIR}/pyenv"
  export PYENV_TEST_DIR="$(mktemp -d "${PYENV_TEST_DIR}.XXX" 2>/dev/null || echo "$PYENV_TEST_DIR")"

  if enable -f "${BATS_TEST_DIRNAME}"/../libexec/pyenv-realpath.dylib realpath 2>/dev/null; then
    export PYENV_TEST_DIR="$(realpath "$PYENV_TEST_DIR")"
  else
    if [ -n "$PYENV_NATIVE_EXT" ]; then
      echo "pyenv: failed to load \`realpath' builtin" >&2
      exit 1
    fi
  fi

  export PYENV_ROOT="${PYENV_TEST_DIR}/root"
  export HOME="${PYENV_TEST_DIR}/home"
  export PYENV_HOOK_PATH="${PYENV_ROOT}/pyenv.d"

  PATH=/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin
  PATH="${PYENV_TEST_DIR}/bin:$PATH"
  PATH="${BATS_TEST_DIRNAME}/../libexec:$PATH"
  PATH="${BATS_TEST_DIRNAME}/libexec:$PATH"
  PATH="${PYENV_ROOT}/shims:$PATH"
  export PATH

  for xdg_var in `env 2>/dev/null | grep ^XDG_ | cut -d= -f1`; do unset "$xdg_var"; done
  unset xdg_var
fi
```

On this line:
  export PYENV_ROOT="${PYENV_TEST_DIR}/root"
  
 Where it sets PYENV_ROOT to a non-existing directory.
 It points to something like `/tmp/pyenv.mbz/root`
 I am trying to *undo* this by setting PYENV_ROOT to what it probably was originally.